app/src/http_error.h

```diff
@@ -3,57 +3,140 @@
 
 #pragma once
 
+#include "cbor.h"
 #include "tracing.h"
 
+#include <qcbor/qcbor_decode.h>
+#include <qcbor/qcbor_encode.h>
+#include <qcbor/qcbor_spiffy_decode.h>
+#include <stdexcept>
+
 namespace scitt
 {
   struct HTTPError : public std::runtime_error
```

Comment:
Considering that all errors are CBOR in SCRAPI, I suspect we could s/HTTPError/HTTPCBORError/ and skip the flag altogether.

Reply:
These are managed errors that we throw. In some cases our API would return whatever CCF returns and it would make things slightly more awkward if our errors were all CBOR but some other errors were JSON. Now at least the JSON endpoints return JSON errors and CBOR endpoints return CBOR. Some native support of `ser_error` in CCF would be great.

Reply:
That's definitely something we ought to look into, yes.

Reply:
I stuck with having 2 sets of errors, we can revisit later and remove JSON altogether